test/issues/issues-test.js

```diff
@@ -200,4 +200,19 @@ describe("issues", function () {
             assert(firstFake !== secondFake);
         });
     });
+
+    describe("#1398", function () {
+        it("Call order takes into account both calledBefore and calledAfter", () => {
+            var s1 = sinon.spy();
+            var s2 = sinon.spy();
+
+            s1();
+            s2();
+            s1();
+
+            assert.exception(function () {
+                sinon.assert.callOrder(s2, s1, s2);
```

Comment:
Yup, In this test I'm expecting an error because that is not the order those spies are called in.

Basically I'm asserting this feature fails when it should.